gradle.properties

```diff
@@ -11,8 +11,8 @@ loader_version=0.4.8+build.159
 fabric_version=0.3.0+build.207
 modmenu_version=1.5.4-85
 # Mod Properties
-mod_version = 1.3.2
-maven_group = io.github.ladysnake
+mod_version = 1.3.3
+maven_group = com.github.Ladysnake
```

Comment:
why is the maven group changed?

Reply:
Because that is the maven group to which you actually publish, per your readme page.  I noted this when I published to maven local and then gradle could not find the dependency.  If the maven group is actually io.github.ladysnake then the readme should be updated.  (Although what you have is what seems to work.)

Reply:
jitpack allows both, `com.` and `io.`, and it's case-insensitive.